### DISCUSSION TITLE
ESP32-POE-ISO

### DISCUSSION THREAD

**Discussion by hweiss24** [tbnobody/OpenDTU#152] [Q&A] [answered]
Hello everybody, i have a simple question. Does also fit this board: [ESP32-POE-ISO](https://www.olimex.com/Products/IoT/ESP32/ESP32-POE-ISO/open-source-hardware)? It seems that is the same as ESP32-POE but with 1000 Mbits. I want to run the OpenDTU with LAN no WLAN ;) Thanks...

**Comment by tbnobody** [comment, marked as answer]
The difference between ESP32-POE and ESP32-POE-ISO is the galvanic insulation. Speed etc. is the same. For development purposes I also used the ESP32-POE-ISO board. So I can confirm that this board is working :)

**Comment by hweiss24** [comment]
Perfect, Thank you ;)

**Comment by raccoon0815** [comment]
Hi, I'm trying to get openDTU running on two Olimex ESP32-PoE-ISO.
Flashing and access via WLAN works.
But the Ethernet port via RJ45 shows no link signal.

i tried

to use power via PoE
to use power via USB and PoE
to use power via USB and directly link to the switch
to use power via lipo connector and directly link to the switch
in every way ich have power and WIFI but RJ45-Ethernet doesn't work.

if i use this example (Arduiuno IDE) the RJ45 link is up and running with 100Mbps
https://github.com/OLIMEX/ESP32-POE/blob/master/SOFTWARE/ARDUINO/ESP32_PoE_MQTT-Azure_ETH_WiFi/ESP32_PoE_MQTT-Azure_ETH_WiFi.ino

did I miss something?

do I have to enter parameters when compiling so that RJ45 is used?

**Reply by tbnobody** [reply]
Did you use the right build environment? It seems that you used the generic environment which does not include ethernet support. You should use the "olimex_esp32_poe" environment

**Reply by raccoon0815** [reply]
thanks for the hint....

I found it:  default_envs = olimex esp32 poe

ESP32-POE now lights up like a Christmas tree

sorry for the stupid question... next time i'll read more carefully before i ask

**Reply by tbnobody** [reply]
You don't have to edit the platformio.ini. You can also switch the environment in the status bar of vscode

**Reply by raccoon0815** [reply]
wow cool :-)
